### PR TITLE
Marine birds example added

### PR DESCRIPTION
**Pull request by SeritavdW**
Hi Guys,
This commit includes an ENV-DATA example for marine birds and created in collab with Dr. Eric Woehler, Lead PI for the seabird survey project on the CSIRO’s RV Investigator (dataset origin).

### CODE REVIEW

File: examples.md [hunk 1]
```diff
@@ -237,8 +237,61 @@ And finally there is the MeasurementOrFact extension table, which has attributes
...
+|-------------------------------------------|---------------------------------|--------------------------|---------------------|
+| urn:lsid:marinespecies.org:taxname:343991 | Morus serrator                  | (Gray,1843)              | Australasian Gannet |
+| urn:lsid:marinespecies.org:taxname:212648 | Pachyptila turtur               | (Kuhl,1820)              | Fairy Prion         |
+| urn:lsid:marinespecies.org:taxname:707545 | Chroicocephalus novaehollandiae | Stephens,1826            | Silver Gull         |
+
+**extendedMeasurementOrFact (eMoF) extension:**
+
+As shown in previous examples, the MeasurementOrFact extension table contains abiotic meaasurements or facts corresponding to an occurrence / sighting. Individual sightings and abiotic measurements are linked with `occurrenceID`. In the example dataset, the ENV-DATA consist of measurements taken during the moorings deployment at the SOTS site, at the time of the marine bird sightings. In addition to NERC vocabulary terms, authors also referenced the Australian Ocean Data Network (AODN) Discovery Parameter Vocabulary for *Depth* and *Sea Surface Temperature* as `measurementType`.
```
Comment (pieterprovoost) [inline examples.md:274]: Typo in meaasurements.

File: examples.md [hunk 2]
```diff
@@ -237,8 +237,61 @@ And finally there is the MeasurementOrFact extension table, which has attributes
...
+
+| occurrenceID     | measurementID                     | measurementType                 | measurementTypeID                                           |
+|------------------|-----------------------------------|---------------------------------|-------------------------------------------------------------|
+| in2017_v02_00998 | in2017_v02_00998-depth            | Depth (m)                       | http://vocab.aodn.org.au/def/discovery_parameter/entity/574 |
```
Comment (pieterprovoost) [inline examples.md:278]: I suggest changing Depth to Sea-floor depth to avoid confusion.

Comment (pieterprovoost) [inline examples.md:278]: Any chance you can add the NERC equivalents for the AODN terms? These can be additional MoF records.

**Review by pieterprovoost — CHANGES_REQUESTED** [review]
Made a few minor suggestions.

**Comment by SeritavdW** [comment]
thanks Pieter!

**Pull request merged**